azure-specialized-workloads/oracle/recommendations.yaml

```diff
@@ -0,0 +1,201 @@
+      url: "https://docs.oracle.com/en/database/oracle/oracle-database/21/haovw/db-azure1.html#GUID-91572193-DF8E-4D7A-AF65-7A803B89E840"
+
+- description: Implement a backup and restore strategy for Oracle databases to meet your workload requirements
+  aprlGuid: 0583239a-dfb5-44d4-94db-804bfc8e3bd1
+  recommendationTypeId: null
+  recommendationControl: Business Continuity
+  recommendationImpact: High
+  recommendationResourceType: Specialized.Workload/oracle
+  recommendationMetadataState: Active
+  longDescription: |
+    When you configure automatic backup to Autonomous Recovery Service or Object Storage Service in OCI, backup copies provide additional protection. Oracle Recovery Manager (RMAN) validates cloud database backups for any physical corruptions.
+  potentialBenefits: Provide workload data protection.
+  pgVerified: false
+  publishedToLearn: false
+  automationAvailable: no
+  tags: null
+  learnMoreLink:
+    - name: Business continuity and disaster recovery considerations for Oracle Database@Azure
+      url: "https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/scenarios/oracle-iaas/oracle-disaster-recovery-oracle-database-azure"
+    - name: Learn about Oracle Maximum Availability Architecture for Oracle Database@Azure
+      url: "https://docs.oracle.com/en/solutions/oracle-maa-db-at-azure/#GUID-7723E2B1-9588-40BC-88BE-44637B1AF0D9"
+
+- description: Scale up the VM cluster based on the workload requirement
+  aprlGuid: 2322a597-a6af-4c3e-a1b1-d1b1ddead508
+  recommendationTypeId: null
+  recommendationControl: Scalability
+  recommendationImpact: Medium
+  recommendationResourceType: Specialized.Workload/oracle
+  recommendationMetadataState: Active
+  longDescription: |
+    Oracle RAC is ideal for high-volume applications or consolidated environments where scalability and the ability to dynamically add or re-prioritize capacity across more than a single server are required. An individual database may have instances running on one or more nodes of a cluster. Similarly, a database service may be available on one or more database instances. Additional nodes, database instances, and database services can be provisioned online.
+    Auto-scaling is not avaialble on ODAA.
+    Be mindful that the capacity of Vm cluster is deployed according to the workload requirements.
+    If scaling up is needed, scale up the number of OCPUs on the VM cluster, if there is capacity available on the Exa infra.
+  potentialBenefits: Meet workload scalability requirements.
+  pgVerified: false
+  publishedToLearn: false
+  automationAvailable: no
+  tags: null
+  learnMoreLink:
+    - name: Learn about Oracle Maximum Availability Architecture for Oracle Database@Azure
+      url: "https://docs.oracle.com/en/solutions/oracle-maa-db-at-azure/#GUID-7723E2B1-9588-40BC-88BE-44637B1AF0D9"
+
+- description: Plan and implement IP addressing strategy to meet current and future requirements
```

Comment:
This appears to be more architectural guidance than a configuration for the Oracle resource that can be configured to improve the resiliency of the Oracle resource deployment. As such, wouldn't this better be addressed through WAF guidance?

Reply:
The recommendation is agreed by relevant teams and as such we would like to keep it

Reply:
Agree with @ehaslett this reads as a generic best practice rather than an actionable reliability recommendation.

Reply:
@basimolimajeed , due to the APRL requirements @ehaslett mentions [above](https://github.com/azure/azure-proactive-resiliency-library-v2/pull/463#discussion_r1819500407), this should be moved to WAF.

Reply:
being discussed